### PR TITLE
add support for the html CODE element

### PR DESCRIPTION
**Pull request by joshmarinacci**
This adds basic support for the CODE element.  CODE is part of the HTML 5 spec and it has semantic meaning, so I think it should be supported by default.

### CODE REVIEW

File: src/quill.coffee [hunk 1]
```diff
@@ -197,9 +197,9 @@ class Quill extends EventEmitter2
     delta.ops.push({ delete: this.getLength() })
     this.updateContents(delta, source)
 
-  setHTML: (html, source = Quill.sources.API) ->
+  setHTML: (html, shouldStripWhitespace, source = Quill.sources.API) ->
```
Comment (willrowe) [inline src/quill.coffee]: This is a BC break.

**Comment by willrowe** [comment]
You should probably just have one feature in this PR, since this is titled as adding the `code` tag I would remove the strip whitespace functionality. Also what's up with the `.idea` directory?

**Comment by joshmarinacci** [comment]
Ah, crud. this push request was supposed to have just the code patch, not the .idea or stripping stuff. I'll fix it.

**Comment by joshmarinacci** [comment]
I just sent a new pull request.

**Pull request closed without merging**